src/arrayMethodSort.js

```diff
@@ -4,9 +4,39 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = compareString) {
+    let count = 1;
```

Comment:
if you use `do...while` loop instead of while - you won't need to set 1 as initial for `count`.
Because 1 looks like some magic number here, we should avoid this